public/app/features/alerting/unified/components/contact-points/ContactPoints.tsx

```diff
@@ -62,17 +62,32 @@ import {
 } from './useContactPoints';
 import { ContactPointWithMetadata, getReceiverDescription, isProvisioned, ReceiverConfigWithMetadata } from './utils';
 
-enum ActiveTab {
-  ContactPoints,
-  NotificationTemplates,
+export enum ActiveTab {
+  ContactPoints = 'contact_points',
+  NotificationTemplates = 'templates',
 }
 
 const DEFAULT_PAGE_SIZE = 10;
 
+const useTabQueryParam = () => {
+  const [queryParams] = useURLSearchParams();
+  return useMemo(() => {
+    const queryParam = queryParams.get('tab');
+
+    if (!queryParam || !Object.values(ActiveTab).map(String).includes(queryParam)) {
+      return ActiveTab.ContactPoints;
+    }
+
+    return queryParam || ActiveTab.ContactPoints;
+  }, [queryParams]);
+};
+
 const ContactPoints = () => {
   const { selectedAlertmanager } = useAlertmanager();
-  // TODO hook up to query params
-  const [activeTab, setActiveTab] = useState<ActiveTab>(ActiveTab.ContactPoints);
+  const [queryParams, setQueryParams] = useURLSearchParams();
+  const activeTabVal = useTabQueryParam();
```

Comment:
How about keeping the active tab state only in url and changing this hook like for example
```
const [activeTab, setActiveTab] = useTabQuery()
```
Then, we could remove the line below, and make the query parameter our only source of truth. 
It should also allow us to remove the `useEffect` below